.devcontainer/devcontainer.json

```diff
@@ -38,10 +38,10 @@
 	"mounts": [
 		"source=projectname-bashhistory,target=/commandhistory,type=volume",
 		"source=${localEnv:HOME}/.ssh,target=/root/.ssh,type=bind,consistency=cached,readonly",
-		"source=${localEnv:HOME}/.aws,target=/root/.aws,type=bind,consistency=cached,readonly",
-		"source=${localEnv:HOME}/.darwin,target=/root/.darwin,type=bind,consistency=cached,readonly",
+		// "source=${localEnv:HOME}/.aws,target=/root/.aws,type=bind,consistency=cached,readonly",
+		// "source=${localEnv:HOME}/.darwin,target=/root/.darwin,type=bind,consistency=cached,readonly",
 		"source=${localEnv:HOME}/.gitconfig,target=/root/.gitconfig,type=bind,consistency=cached,readonly",
-		"source=/data,target=/data,type=bind,consistency=cached",
+		// "source=/data,target=/data,type=bind,consistency=cached",
```

Comment:
Good call -- can you just delete these lines instead of commenting?

Reply:
done!